src/game.zig

```diff
@@ -904,6 +1002,9 @@ pub fn update(deltaTime: f64) void { // MARK: update()
 						movementSpeed = @max(movementSpeed, 5.5);
 						movementDir[2] += 5.5;
 					}
+				} else if(Player.super.climbing) {
+					movementSpeed = @max(movementSpeed, Player.currentClimbSpeed*0.7);
+					movementDir += up*@as(Vec3d, @splat(walkingSpeed));
```

Comment:
`movementDir` is normalized.
This has the side effect off slowing you down even if you just walk alongside a climbable block.
Up direction should be handled separately here.